src/models/db/account.js

```diff
@@ -109,7 +116,13 @@ PersistentModelMixin(Account, sequelize, {
   password: Sequelize.STRING,
   public_key: Sequelize.TEXT,
   is_admin: Sequelize.BOOLEAN,
-  is_disabled: Sequelize.BOOLEAN
-})
+  is_disabled: Sequelize.BOOLEAN,
+  fingerprint: Sequelize.STRING
```

Comment:
I think you also have to add this to the schema in five-bells-shared and add a migration